webpack.config.js

```diff
@@ -18,15 +18,20 @@ const developmentEntries = isDevelopment
 
 const plugins = [
   new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en/),
-  new CopyWebpackPlugin([
-    {
-      from: 'static',
-      to: 'static',
-    },
-  ]),
+  new webpack.ProvidePlugin({
+    process: 'process/browser',
+  }),
```

Comment:
Vfile (which is a dependency of a dependency of a dependency) requires process to be shimmed in the browser for it to work.